libs/components/forms/src/lib/modules/input-box/input-box.component.ts

```diff
@@ -241,6 +241,13 @@ export class SkyInputBoxComponent
         this.hintText,
       );
 
+      this.#adapterService.updateDescribedBy(
+        this.inputRef,
+        this.errorId,
+        'errorId', // Passing a string here to force the aria-describedby attribute to include the errorId.
```

Comment:
We talked offline, but it might be worth wrapping both the errors and the hint text elements in a single DIV with an ID. That way, we're only worrying about one `aria-describedby` element instead of two.
```
<input [aria-describedby]="fieldDescriptors.id" />

<div skyId #fieldDescriptors>
  <sky-form-errors />
  <hint-text />
</div>
```

Reply:
Also, I noticed that this component's `hintText` setter [emits to the `ariaDescribedBy` observable](https://github.com/blackbaud/skyux/blob/main/libs/components/forms/src/lib/modules/input-box/input-box.component.ts#L140) (which is passed through to the [`SkyInputBoxHostService.ariaDescribedBy`](https://github.com/blackbaud/skyux/blob/main/libs/components/forms/src/lib/modules/input-box/input-box-host.service.ts#L23-L25) and [subscribed to by the country field](https://github.com/blackbaud/skyux/blob/main/libs/components/lookup/src/lib/modules/country-field/country-field.component.ts#L552). 

We need to figure out an elegant way to emit the `ariaDescribedBy` observable when errors change as well.

Reply:
Like we talked about, not touching `aria-describedby` at this point.